### DISCUSSION TITLE
Bug: showng selection in inactive editor

### DISCUSSION THREAD

**Discussion by evskorobogatij** [facebook/lexical#5639] [Q&A]
I need to show selection in inactive editor

Lexical version: 0.7.6

## Steps To Reproduce

1. In example sandbox type some text in editor
2. Select fragment in text
3. Click on 'type...' input in toolbar
4. We lost selection in editor.... but as we see in 'selection' object selection exist. 

How display selection in inactive editor? 

Link to code example:

https://codesandbox.io/s/young-microservice-jbfn04?file=/src/plugins/ToolbarPlugin.js

## The current behavior
lost selection if editor lost focus

## The expected behavior
Show selection on inactive editor

**Comment by aquarius-wing** [comment]
Maybe you should store the "lastSelection" in some pulgin of lexical?

**Comment by evskorobogatij** [comment]
I apply styles for imitate display selection

```
editor.update(() => {
        //
        const selection = $getSelection()
        if ($isRangeSelection(selection)) {
          $patchStyleText(selection, { color: 'white', 'background-color': '#0078d7' })
        }
      })
```


How to cancel last action in editor?

**Comment by zurfyx** [comment]
This is a behavior that is not supported by the browser but we now provide utilities for this: https://github.com/facebook/lexical/pull/5153. This will allow you to mimic the selection as if it was selected by the browser.
